python/paddle/fluid/executor.py

```diff
@@ -432,12 +432,12 @@ def handler(self, res_dict):
 class Executor(object):
     """
     An Executor in Python, supports single/multiple-GPU running,
-    and single/multiple-CPU running. When construction the Executor,
-    the device is required.
+    and single/multiple-CPU running.
 
     Args:
-        place(fluid.CPUPlace()|fluid.CUDAPlace(n)): This parameter represents
-            the executor run on which device.
+        place(fluid.CPUPlace()|fluid.CUDAPlace(n)|None): This parameter represents
```

Comment:
This parameter represents which device the executor run on

Reply:
Done.

Reply:
My fault, here should be "the executor runs on". Notice the "run" -> "runs"

Reply:
Done.